.rubocop.yml

```diff
@@ -7,7 +7,7 @@ AllCops:
 
 # Baseline code complexity metrics - we should try and reduce these over time
 Metrics/AbcSize:
-  Max: 47
+  Max: 71.5
```

Comment:
Why do you prefer disabling the cop?  imo it's easier to track the metric size in `.rubocop.yml` than tracking multiple cop disables.